### PR TITLE
Adds the "Supermatter Surge" random event from cit

### PR DESCRIPTION
**Pull request by UselessTheremin**
### Intent of your Pull Request

ports https://github.com/Citadel-Station-13/Citadel-Station-13/pull/14307

### Why is this change good for the game?

it is fun to stress test an otherwise extremely safe engine and give it a bit more gameplay

### Briefly describe your PR and the impacts of it, in layman's terms. 

adds the supermatter surge random event which, when triggered, increases the power of the SM for a time.

### What general grouping does this PR fall under? 

engineering change, new random event.

:cl:  
rscadd: adds a new random event which increases the power level of the sm for a bit
/:cl:

### CODE REVIEW

**Review by redmoogle — APPROVED** [review]
Code looks fine although one important note.

Matter_power is power over time while power is immediate power

**Comment by MegaEmpirical** [comment]
I like it.
